src/style.css

```diff
@@ -0,0 +1,37 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stars__star {
+  margin: 0 4px;
+  width: 16px;
+  height: 16px;
+  background-size: cover;
+  background-image: url(images/star.svg);
+}
+
+.stars--1 .stars__star:nth-child(-n+1) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--2 .stars__star:nth-child(-n+2) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--3 .stars__star:nth-child(-n+3) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--4 .stars__star:nth-child(-n+4) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--5 .stars__star:nth-child(-n+5) {
+  background-image: url(images/star-active.svg);
+}
```

Comment:
Please pay more attention to the task [checklist](https://github.com/mate-academy/layout_stars/blob/master/checklist.md)

![image](https://github.com/mate-academy/layout_stars/assets/80700247/b782b397-44ee-4feb-b07a-64c3a3db318c)